makefile

```diff
@@ -0,0 +1,15 @@
+GIT_TAG=v2.0.8
+REPO_NAME=quay.io/coreos/clair-jwt
+SUBSCRIPTION_KEY=
+centos7-build-env: centos7-build-env.Dockerfile
+	echo "Building Clair build environment with tag: '${REPO_NAME}:${GIT_TAG}-centos7-build-env'"
+	docker build -f centos7-build-env.Dockerfile -t ${REPO_NAME}:${GIT_TAG}-centos7-build-env . --build-arg GIT_TAG=${GIT_TAG}
+centos7: centos7.Dockerfile 
+	echo "Building CentOS based image with tag: '${REPO_NAME}:${GIT_TAG}-centos7'"
+	docker build -f centos7.Dockerfile -t ${REPO_NAME}:${GIT_TAG}-centos7 . --build-arg GIT_TAG=${GIT_TAG}
+rhel7: rhel7.Dockerfile
+	echo "Building RHEL based image with tag: '${REPO_NAME}:${GIT_TAG}-rhel7', with path to subscription key '${SUBSCRIPTION_KEY}' (PEM format)"
+	docker build --squash -f rhel7.Dockerfile -t ${REPO_NAME}:${GIT_TAG}-rhel7 . --build-arg GIT_TAG=${GIT_TAG} --build-arg SUBSCRIPTION_KEY=${SUBSCRIPTION_KEY}
```

Comment:
might want to add a comment here or on the Dockerfile that we have to squash to avoid including the subscription key used to build the image